kernel/arch/dreamcast/include/dc/sound/stream.h

```diff
@@ -238,7 +238,7 @@ void snd_stream_queue_disable(snd_stream_hnd_t hnd);
 */
 void snd_stream_queue_go(snd_stream_hnd_t hnd);
 
-/** \brief  Start a stream.
+/** \brief  Start PCM 16-bit stream.
```

Comment:
```suggestion
/** \brief  Start a 16-bit PCM stream.
```

I feel like this flows better...